tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py

```diff
@@ -38,8 +38,26 @@
 ]
 term_input_for_missing_terms = [
     # test_name, num_res, reco_json_schema, expected_duration_in_hours, increment_end_time_by
-    ("medium_term_test_192_data_points_non_contiguous", 192, medium_term_list_reco_json_schema, 48.0),
-    ("long_term_test_768_data_points_non_contiguous", 768, long_term_list_reco_json_schema, 192.0),
+    # contiguous scenarios
+    ("no_term_min_recomm", 1, None, 0, 0),
+    ("all_terms_min_recomm", 768, list_reco_json_schema, 192.0, 0),
+    ("only_short_term_min_recomm", 2, list_reco_json_schema, 0.5, 0),
+    ("only_medium_term_min_recomm", 192, medium_term_list_reco_json_schema, 48.0, 0),
+    ("only_long_term_min_recomm", 768, long_term_list_reco_json_schema, 192.0, 0),
+    ("short_and_medium_term_min_recomm", 192, list_reco_json_schema, 48.0, 0),
+    ("short_and_long_term_min_recomm", 768, list_reco_json_schema, 192.0, 0),
```

Comment:
Schema's need to be updated here. Existing schema's in the test may not work for all cases, we need to discuss on how to validate these and make changes later.

Reply:
Updated and added new schema files